lib/xapi-stdext-threads/threadext.ml

```diff
@@ -45,7 +45,7 @@ module Alarm = struct
     | None -> assert false
     | Some (pipe_in, pipe_out) ->
       while Thread.wait_timed_read pipe_in 0. do
-        ignore (Unix.read pipe_in " " 0 1)
+        ignore (Unix.read pipe_in (Bytes.create 1) 0 1)
```

Comment:
We could create these bytes values globally once rather than on demand. However, in this context it does not matter, I suppose.

Reply:
True. I opted for the `(Bytes.create 1)` becaused it was already used as a pattern somewhere else in the library